index.bs

```diff
@@ -828,7 +828,12 @@ sequence&lt;{{WebTransportHash}}&gt; |serverCertificateHashes|, run these steps.
 1. Let |request| be a new [=/request=] whose [=request/URL=] is |url|, [=request/client=] is
    |client|, [=request/policy container=] is |client|'s
    [=environment settings object/policy container=], [=request/destination=] is an empty string,
-   and [=request/origin=] is |origin|.
+   [=request/origin=] is |origin| and [=request/redirect mode=] is "error". 
```

Comment:
`request` appears only used for CSP checks etc, not passed into obtain a connection. So check where redirect mode is handled in fetch